scipy/_lib/tests/test_public_api.py

```diff
@@ -330,16 +334,33 @@ def check_importable(module_name):
                           ('scipy.interpolate.ndgriddata', None),
                           ('scipy.interpolate.polyint', None),
                           ('scipy.interpolate.rbf', None),
+                          ('scipy.io.harwell_boeing', None),
+                          ('scipy.io.idl', None),
+                          ('scipy.io.mmio', None),
+                          ('scipy.io.netcdf', None),
+                          ('scipy.io.arff.arffread', 'arff'),
+                          ('scipy.io.matlab.byteordercodes', 'matlab'),
+                          ('scipy.io.matlab.mio_utils', 'matlab'),
+                          ('scipy.io.matlab.mio', 'matlab'),
+                          ('scipy.io.matlab.mio4', 'matlab'),
+                          ('scipy.io.matlab.mio5_params', 'matlab'),
+                          ('scipy.io.matlab.mio5_utils', 'matlab'),
+                          ('scipy.io.matlab.mio5', 'matlab'),
+                          ('scipy.io.matlab.miobase', 'matlab'),
+                          ('scipy.io.matlab.streams', 'matlab'),
                           ('scipy.linalg.basic', None),
                           ('scipy.linalg.decomp', None),
                           ('scipy.linalg.decomp_cholesky', None),
                           ('scipy.linalg.decomp_lu', None),
                           ('scipy.linalg.decomp_qr', None),
                           ('scipy.linalg.decomp_schur', None),
                           ('scipy.linalg.decomp_svd', None),
+                          ('scipy.linalg.flinalg', None),
```

Comment:
originally I didn't touch `flinalg` because, after the next release, there would be nothing left in it. In which case it seems logical at that point just to remove it however officially it is slated for 2.0.0 so there is an awkward contradiction. Any thoughts?

Reply:
@h-vetinari do you have an opinion here?

Reply:
@h-vetinari friendly ping here as we are a few days away from branching

@j-bowhay is there any harm in making the change here even if we do end up removing `flinalg`?

Reply:
Plan on merging tomorrow if there are no further comments